php/commands/search-replace.php

```diff
@@ -61,6 +62,9 @@ class Search_Replace_Command extends WP_CLI_Command {
 	 * : Write transformed data as SQL file instead of saving replacements to
 	 * the database. If <file> is not supplied, will output to STDOUT.
 	 *
+	 * [--export_insert_size[=<rows>]]
```

Comment:
This should be `[--export_insert_size=<rows>]`

Note that it shouldn't ever be used as a flag.